### DISCUSSION TITLE
How do I know when a new file is uploaded to the server?

### DISCUSSION THREAD

**Discussion by f2065** [rejetto/hfs#821] [Ideas]
Is there a way to know that a new file has been uploaded to the server?
So that the server administrator quickly knows that someone has uploaded a file.
For example, maybe add another log file, where there will only be information about the upload in a simple form?
“IP – date/time – filesize – fullfilename”
Then the fact that this log appears would signal the appearance of the file.
Maybe there are other ways?

**Comment by rejetto** [comment]
a user could go the admin-panel > log, and filter the column "notes" with "upload".
a separate software could call GET localhost/~/api/get_log { file: 'log' } and then filter by notes.
a plugin could subscribe event 'log' and then filter by notes.
